runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -11131,11 +11041,11 @@ J9::Z::TreeEvaluator::VMnewEvaluator(TR::Node * node, TR::CodeGenerator * cg)
       if (isArray)
          {
          if ( comp->compileRelocatableCode() && opCode == TR::anewarray)
-         genInitArrayHeader(node, iCursor, isVariableLen, classAddress, classReg, resReg, zeroReg,
-               enumReg, dataSizeReg, temp1Reg, litPoolBaseReg, conditions, cg);
+         genInitArrayHeader(node, iCursor, isVariableLen, classAddress, classReg, resReg,
```

Comment:
Seems like for anewarray and relocatable code, only parameter that is passed is classReg, can we shrink this to one statement with conditional assignment ?